core/src/test/java/hudson/util/XStream2Test.java

```diff
@@ -23,24 +23,29 @@
  */
 package hudson.util;
 
-import static org.junit.Assert.*;
-
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableMap;
-import com.thoughtworks.xstream.XStreamException;
+import static org.junit.Assert.assertEquals;
```

Comment:
Sorry, I made this pull request accidently.

Am 01.07.15 KW26 um 20:33 schrieb Kanstantsin Shautsou:
>> @@ -23,24 +23,29 @@
>>   */
>>  package hudson.util;
>>  
>> -import static org.junit.Assert.*;
>> -
>> -import com.google.common.collect.ImmutableList;
>> -import com.google.common.collect.ImmutableMap;
>> -import com.thoughtworks.xstream.XStreamException;
>> +import static org.junit.Assert.assertEquals;
> No need in replacing static import with `*`
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/jenkinsci/jenkins/pull/1752/files#r33708954